master/internal/user/authz_iface.go

```diff
@@ -59,13 +58,9 @@ type UserAuthZ interface {
 
 	// GET /api/v1/users/setting
 	CanGetUsersOwnSettings(ctx context.Context, curUser model.User) error
-	// PATCH /api/v1/users/setting
-	CanCreateUsersOwnSetting(
-		ctx context.Context, curUser model.User, setting model.UserWebSetting,
-	) error
 	// POST /api/v1/users/setting
-	CanOverwriteUsersOwnSettings(
-		ctx context.Context, curUser model.User, settings []*userv1.UserWebSetting,
```

Comment:
this would need a corresponding ee piece as well. sometimes it might be easier for rbac-touching changes to start in EE and get ported back here once all is good.